Document/0x08-Testing-Tools.md

```diff
@@ -2,184 +2,164 @@
+- [Cydia Substrate: Introspy-Android](https://github.com/iSECPartners/Introspy-Android "Introspy Android") - Blackbox tool to help understand what an Android application is doing at runtime and assist in the identification of potential security issues.
+- [Drozer](https://www.mwrinfosecurity.com/products/drozer/ "Drozer") - Drozer allows you to search for security vulnerabilities in apps and devices by assuming the role of an app and interacting with the Dalvik VM, other apps' IPC endpoints and the underlying OS.
+- [Inspeckage](https://github.com/ac-pm/Inspeckage "Inspeckage") - Inspeckage is a tool developed to offer dynamic analysis of Android apps. By applying hooks to functions of the Android API, Inspeckage will help you understand what an Android application is doing at runtime.
+- [logcat-color](https://github.com/marshall/logcat-color "Logcat color") - A colorful and highly configurable alternative to the adb logcat command from the Android SDK.
+- [VirtualHook](https://github.com/rk700/VirtualHook "VirtualHook") - VirtualHook is a hooking tool for applications on Android ART(>=5.0). It's based on VirtualApp and therefore does not require root permission to inject hooks.
+- [Xposed Framework](https://forum.xda-developers.com/xposed/xposed-installer-versions-changelog-t2714053 "Xposed Framework") - Xposed framework enables you to modify the system or application aspect and behavior at runtime, without modifying any Android application package(APK) or re-flashing.
 
 #### Bypassing Root Detection and Certificate Pinning
 
-- [Xposed Module: Just Trust Me](https://github.com/Fuzion24/JustTrustMe "Xposed Module: Just Trust Me") - Xposed Module to bypass SSL certificate pinning.
-- [Xposed Module: SSLUnpinning](https://github.com/ac-pm/SSLUnpinning_Xposed "Xposed Module: SSLUnpinning") - Android Xposed Module to bypass SSL certificate validation (Certificate Pinning)).
 - [Cydia Substrate Module: Android SSL Trust Killer](https://github.com/iSECPartners/Android-SSL-TrustKiller "Cydia Substrate Module: Android SSL Trust Killer") - Blackbox tool to bypass SSL certificate pinning for most applications running on a device.
 - [Cydia Substrate Module: RootCoak Plus](https://github.com/devadvance/rootcloakplus "Cydia Substrate Module: RootCoak Plus") - Patch root checking for commonly known indications of root.
-- [Android-ssl-bypass](https://github.com/iSECPartners/android-ssl-bypass "Android-ssl-bypass") - an Android debugging tool that can be used for bypassing SSL, even when certificate pinning is implemented, as well as other debugging tasks. The tool runs as an interactive console.
-- [Frida CodeShare](https://codeshare.frida.re/ "Frida CodeShare") - The Frida CodeShare project is comprised of developers from around the world working together with one goal - push Frida to its limits in new and innovative ways.
-
-#### Security Libraries
```

Comment:
Why did this sectionhave to go?

Reply:
I made a generic section for dynamic instrumentation so Frida Codeshare can be part of that. Doesnt make sense to list it in iOS and Android 
https://github.com/iSECPartners/android-ssl-bypass is from 2013 without any update since